cmd/glasskube/cmd/describe.go

```diff
@@ -242,9 +342,60 @@ func nameAndDescription(manifest *v1alpha1.PackageManifest) string {
 	}
 	return bld.String()
 }
+func printJSON(ctx context.Context,
+	pkg *v1alpha1.Package,
+	manifest *v1alpha1.PackageManifest,
+	latestVersion string,
+	repos []v1alpha1.PackageRepository) {
+	output := map[string]interface{}{
+		"Package":         nameAndDescription(manifest),
+		"Version":         version(pkg, latestVersion),
+		"Status":          status(pkg),
+		"AutoUpdate":      clientutils.AutoUpdateString(pkg, "Disabled"),
+		"Entrypoints":     entrypointsAsMap(manifest),
+		"Dependencies":    dependenciesAsMap(manifest),
+		"Repositories":    repositoriesAsMap(pkg, repos),
+		"References":      referencesAsMap(ctx, pkg, manifest),
+		"LongDescription": strings.TrimSpace(manifest.LongDescription),
+		"Configuration":   valueConfigurationsAsMap(pkg.Spec.Values),
+	}
+	jsonOutput, err := json.MarshalIndent(output, "", "  ")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "❌ Could not marshal JSON output: %v\n", err)
+		cliutils.ExitWithError()
+	}
+	fmt.Println(string(jsonOutput))
+}
+
+func printYAML(
+	ctx context.Context,
+	pkg *v1alpha1.Package,
+	manifest *v1alpha1.PackageManifest,
+	latestVersion string,
+	repos []v1alpha1.PackageRepository) {
+	output := map[string]interface{}{
+		"Package":         nameAndDescription(manifest),
+		"Version":         version(pkg, latestVersion),
+		"Status":          status(pkg),
+		"AutoUpdate":      clientutils.AutoUpdateString(pkg, "Disabled"),
+		"Entrypoints":     entrypointsAsMap(manifest),
+		"Dependencies":    dependenciesAsMap(manifest),
+		"Repositories":    repositoriesAsMap(pkg, repos),
+		"References":      referencesAsMap(ctx, pkg, manifest),
+		"LongDescription": strings.TrimSpace(manifest.LongDescription),
+		"Configuration":   valueConfigurationsAsMap(pkg.Spec.Values),
+	}
```

Comment:
Please start all keys with a lowercase character, since that is more common when using JSON and YAML